tests/test_benchmark_KPI.py

```diff
@@ -140,15 +140,6 @@ def test_benchmark_Economic_KPI_C2_E2(self, margs):
             sep=",",
             index_col=0,
         )
-        # Define numbers in the csv as int/floats instead of str, but leave row "group" as a string
-        groups = expected_values.loc["group"]
-        # need to transpose the DataFrame before applying the conversion and retranspose after
-        # the conversion because it does not follow the tidy data principle
-        # see https://en.wikipedia.org/wiki/Tidy_data for more info
-        expected_values = expected_values.T.apply(
-            pd.to_numeric, errors="ignore", downcast="integer"
-        ).T
-        expected_values.loc["group"] = groups
         expected_values.loc[FLOW] = [0, 0, 0, 0, 0]
```

Comment:
Why don´t you write that value to the original csv as well? No processing in python needed.

Reply:
Removed in 81edacd59ff18008a51dda1dbb8e2a9e078b3b7f.